domain-models-runtime/src/main/java/org/folio/rest/persist/Criteria/Criteria.java

```diff
@@ -114,15 +114,15 @@ private void createFromSnippet() {
       from = new From();
       from.setSnippet(ARRAY_FROM_CLAUSE + "("
         + PostgresClient.DEFAULT_JSONB_FIELD_NAME + GET_JSON_FIELD + arrayField + ")");
-      from.setAsValue(field.get(0).replaceAll("^'|'$", "")); //remove ''
+      from.setAsValue(field.get(0).replace("'", "")); //remove ''
     }
```

Comment:
Not the same .. Eg the original would not remove quote from `a'b` , but the new one will. I believe all quotes should be removed, so OK by me.

Reply:
Ok thanks.